README.md

```diff
@@ -35,6 +35,7 @@ Invoked by our GitHub Actions workflows, including the reusable workflows below.
   ([README](actions/setup-ssh/README.md))
 - [Setup debugger](actions/setup-debugger/action.yaml) for interactive debugging of workflow jobs
   ([README](actions/setup-debugger/README.md))
+- [Run Nextstrain build on AWS Batch](actions/run-nextstrain-build-on-aws-batch/action.yaml)
```

Comment:
`nextstrain/.github/actions/run-nextstrain-build-on-aws-batch` seems a bit wordy and overly specific to a particular implementation. I think we ideally want an action that encapsulates a _concept_ ("do the builds"), not a specific implementation. We want to be able to evolve the internals of the action over time without always changing all its callsites.

What about `nextstrain/.github/actions/build` or `…/run-builds` or `…/pathogen-repo-builds` (akin to the existing `pathogen-repo-ci` shared workflow)?

(The name is an obvious place to attach my point here, but the point is broader and applies to the action at a whole and the interface it ideally presents.)

Reply:
Hmm, I was wanting to be obvious that the build will be run on AWS Batch as opposed to within the GitHub Action itself. Now that I'm thinking about it, we can make the same workflow be able to run either way and just get toggled to AWS by a provided input.

Reply:
Yeah, we should aim for the runtime to be ~transparent.

Reply:
In #44 the name is now `run-build`. I think this is an improvement! and don't want to belabor the point, but the more I think about it the more I think the name should parallel `pathogen-repo-ci` with a name like `pathogen-repo-build`.

I know I suggested `run-builds` (and even `build`), but they feel too generic esp. with what's already a very overloaded term.

Reply:
Addressed in #44